db/seeds.rb

```diff
@@ -12,8 +12,8 @@ def load_data
     load_crops
     load_plant_parts
 
-    # for development environments only
-    if Rails.env.development?
+    # for development and staging environments only
+    if Rails.env.development? || Rails.env.staging?
```

Comment:
@Skud / @pozorvlak : This change made me a bit nervous. Can you confirm this is okay?